pkg/relayer/session/sessiontree.go

```diff
@@ -298,6 +297,6 @@ func (st *sessionTree) StartClaiming() error {
 }
 
 // GetSupplierOperatorAddress returns a CosmosSDK address of the supplier this sessionTree belongs to.
-func (st *sessionTree) GetSupplierOperatorAddress() *cosmostypes.AccAddress {
+func (st *sessionTree) GetSupplierOperatorAddress() string {
```

Comment:
DItto - not sure why we regressed on this.

Reply:
Should have been a string from the beginning. Each call to `GetSupplierOperatorAddress` as followed by `.String()`